americangut/per_category.py

```diff
@@ -26,11 +28,83 @@ def cat_taxa_summaries():
         for col in table.ids():
             if col == 'Unknown':
                 continue
-            cleaned_col = col.split('(')[0].strip().replace(' ', '_')
+            # Clean the string to just the base information of the response
+            # and url compliant names.
+            cleaned_col = col.split('(')[0].strip().replace(
+                ' ', '_').rstrip('+')
             filename = '-'.join([name, cleaned_col]) + '.txt'
 
             with open(join(out_dir, filename), 'w') as f:
                 for otu, val in zip(ids, table.data(col)):
                     if val == 0.0:
                         continue
                     f.write('%s\t%s\n' % (otu, str(val)))
+
+
+def cat_alpha_plots():
```

Comment:
This generates images, and the core function generating the image is tested, so I'm not sure how to unit test this. Do you have suggestions?